bot/exts/filters/antispam.py

```diff
@@ -83,7 +83,11 @@ async def upload_messages(self, actor_id: int, modlog: ModLog) -> None:
         )
 
         # For multiple messages or those with excessive newlines, use the logs API
-        if len(self.messages) > 1 or 'newlines' in self.rules:
+        if any((
+            len(self.messages) > 1,
+            len(self.messages[0].attachments) > 0,
+            self.messages[0].count('\n') > 15
+        )):
```

Comment:
`self.messages` is a dictionary mapping message IDs to message objects, so this doesn't work.
I get this error:
```
bot_1        | 2021-10-05 12:13:33 | bot.utils.scheduling | ERROR | Error in task AntiSpam._process_deletion_context(frozenset({<Member id=493839819168808962 name='wookie' discriminator='4017' bot=False nick=None guild=<Guild id=813116921646022716 name="wookie's test server" shard_id=None chunked=True member_count=4>>})) 139705866738800!
bot_1        | Traceback (most recent call last):
bot_1        |   File "/bot/bot/exts/filters/antispam.py", line 299, in _process_deletion_context
bot_1        |     await deletion_context.upload_messages(self.bot.user.id, self.mod_log)
bot_1        |   File "/bot/bot/exts/filters/antispam.py", line 89, in upload_messages
bot_1        |     len(self.messages[0].attachments) > 0,
bot_1        | KeyError: 0
```